pyresample/test/test_geometry.py

```diff
@@ -2037,6 +2037,32 @@ def test_get_lonlats(self):
         np.testing.assert_allclose(lats[:464, :], lats0)
         np.testing.assert_allclose(lats[464:, :], lats1)
 
+        # check get_lonlats with chunks definition doesn't cause errors and arrays are equal
+        # too many chunks
+        lons_c, lats_c = final_area.get_lonlats(chunks=((2000, 3568, 5568), (464,)))
+        np.testing.assert_array_equal(lons, lons_c)
+        np.testing.assert_array_equal(lats, lats_c)
+        # too few chunk
+        lons_c, lats_c = final_area.get_lonlats(chunks=((5568,), (464,)))
+        np.testing.assert_array_equal(lons, lons_c)
+        np.testing.assert_array_equal(lats, lats_c)
```

Comment:
you could make a new function for all these cases, like:
```python
def _assert_final_area_lon_lat(final_area, lons, lats, chunks):
        lons_c, lats_c = final_area.get_lonlats(chunks=chunks)
        np.testing.assert_array_equal(lons, lons_c)
        np.testing.assert_array_equal(lats, lats_c)
```

Reply:
Done!